### PR TITLE
Add Ukrainian Language Support for Space Acres

### PR DESCRIPTION
**Pull request by Shegyll**
This PR adds the Ukrainian Language Support for the Space Acres interface

### CODE REVIEW

File: res/translations/uk-UA/messages.ftl [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+configuration_button_cancel = Відмінити
+configuration_button_back = Повернутись
+configuration_button_save = Зберегти
+configuration_button_start = Розпочати
+configuration_dialog_button_select = Обрати
+configuration_dialog_button_cancel = Відмінити
+
+running_title = Запущено
+running_node_title = {$chain_name} вузол консенсусу
+running_node_title_tooltip = Натисніть , щоб відкрити в файловому менеджері.
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:96]:
```suggestion
running_node_title_tooltip = Натисніть щоб відкрити в файловому менеджері.
```

File: res/translations/uk-UA/messages.ftl [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+running_node_free_disk_space_tooltip = Вільний дисковий простір: {$size} remaining
+running_node_status_connecting = Підключення до мережі, best block #{$block_number}
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:98]:
```suggestion
running_node_status_connecting = Підключення до мережі, кращий блок #{$block_number}
```

File: res/translations/uk-UA/messages.ftl [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+running_node_status_syncing_speed_no_eta = , {NUMBER($blocks_per_second, minimumFractionDigits: 2, maximumFractionDigits: 2)} блоки/с
+running_node_status_syncing_speed_hours_eta = , {NUMBER($a_blocks_per_second, minimumFractionDigits: 2, maximumFractionDigits: 2)} блоки/с (~{NUMBER($b_hours_remaining, minimumFractionDigits: 2, maximumFractionDigits: 2)} годин залишилось)
+running_node_status_syncing_speed_minutes_eta = , {NUMBER($a_blocks_per_second, minimumFractionDigits: 2, maximumFractionDigits: 2)} блоки/с (~{NUMBER($b_hours_remaining, minimumFractionDigits: 2, maximumFractionDigits: 2)} хвилин залишилось)
+running_node_status_syncing_speed_seconds_eta = , {NUMBER($a_blocks_per_second, minimumFractionDigits: 2, maximumFractionDigits: 2)} блоки/с (~{NUMBER($b_hours_remaining, minimumFractionDigits: 2, maximumFractionDigits: 2)} секунд залишилось)
+running_node_status_syncing =
+    {$sync_kind ->
+        [dsn] Синхронізація з DSN
+        [regular] Звичайна sync
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:106]:
```suggestion
        [regular] Звичайна синхронізація
```

File: res/translations/uk-UA/messages.ftl [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+        *[unknown] Невідомий тип синхронізації {$sync_kind}
+    } #{$best_block_number}/{$target_block}{$sync_speed}
+running_node_status_synced = Синхронізовано, best block #{$best_block_number}
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:109]:
```suggestion
running_node_status_synced = Синхронізовано, кращий блок #{$best_block_number}
```

File: res/translations/uk-UA/messages.ftl [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+status_bar_message_configuration_is_invalid = Конфігурація недійсна: {$error}
+status_bar_message_restart_is_needed_for_configuration = Для того щоб зміни конфігурації вступили в силу, потрібен перезапуск програми
+status_bar_message_failed_to_save_configuration = Не вдалося зберегти зміни конфігурації: {$error}
+status_bar_message_restarted_after_crash = Space Acres автоматично перезапустилася після неочікуваної помилки , перевірте журнали програми та системи для отримання деталей.
+status_bar_button_restart = Перезапустити
+status_bar_button_ok = Ок
+
+about_system_information =
+    Каталог конфігурації: {$config_directory}
+    Каталог даних (including logs): {$data_directory}
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:194]:
```suggestion
    Каталог даних (включаючи журнали): {$data_directory}
```

File: res/translations/uk-UA/messages.ftl [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+running_farmer_farm_farming = Фармінг
+running_farmer_farm_waiting_for_node_to_sync = Очікування вузла для синхронізації
+running_farmer_farm_sector = Сектор {$sector_index}
+running_farmer_farm_sector_up_to_date = Sector {$sector_index}: актуальна версія
+running_farmer_farm_sector_waiting_to_be_plotted = Sector {$sector_index}: очікування плотингу
+running_farmer_farm_sector_about_to_expire = Sector {$sector_index}: наближається до закінчення терміну дії , очікує реплотингу.
+running_farmer_farm_sector_expired = Sector {$sector_index}: Термін дії закінчився, очікує реплотинг.
+running_farmer_farm_sector_downloading = Sector {$sector_index}: завантаження
+running_farmer_farm_sector_encoding = Sector {$sector_index}: кодується
+running_farmer_farm_sector_writing = Sector {$sector_index}: записується
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:158]: `Sector` -> `Сектор`

File: res/translations/uk-UA/messages.ftl [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+        [today] сьогодні
+        [this_week] цього тижня
+        [more_than_a_week] більше тижня
+        *[unknown] невідомо
+    }
+running_farmer_farm_tooltip = Натисніть , щоб відкрити в файловому менеджері
+running_farmer_farm_reward_signatures_tooltip = {$successful_signatures}/{$total_signatures} Успішні підписи винагороди, перегляньте деталі ферми, щоб побачити більше інформації.
+running_farmer_farm_auditing_performance_tooltip = Аудит ефективності: середній час {NUMBER($a_average_time, minimumFractionDigits: 2, maximumFractionDigits: 2)}s, ліміт часу {NUMBER($b_time_timit, minimumFractionDigits: 2, maximumFractionDigits: 2)}s
+running_farmer_farm_proving_performance_tooltip = Підтвердження ефективності : середній час {NUMBER($a_average_time, minimumFractionDigits: 2, maximumFractionDigits: 2)}s, ліміт часу {NUMBER($b_time_timit, minimumFractionDigits: 2, maximumFractionDigits: 2)}s
+running_farmer_farm_non_fatal_error_tooltip = При фармінгу сталася помилка , яка була усунена. Перегляньте журнали для отримання додаткової інформації: {$error}
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:128]: Extra whitespaces and unnecessary coma:
```suggestion
running_farmer_farm_proving_performance_tooltip = Підтвердження ефективності: середній час {NUMBER($a_average_time, minimumFractionDigits: 2, maximumFractionDigits: 2)}s, ліміт часу {NUMBER($b_time_timit, minimumFractionDigits: 2, maximumFractionDigits: 2)}s
running_farmer_farm_non_fatal_error_tooltip = При фармінгу сталася помилка яка була усунена. Перегляньте журнали для отримання додаткової інформації: {$error}
```

File: res/translations/uk-UA/messages.ftl [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+running_farmer_next_reward_estimate =
+    Наступна винагорода: {$eta_string ->
+        [any_time_now] у будь-який момент
+        [less_than_an_hour] менше години
+        [today] сьогодні
+        [this_week] цього тижня
+        [more_than_a_week] більше тижня
+        *[unknown] невідомо
+    }
+running_farmer_farm_tooltip = Натисніть , щоб відкрити в файловому менеджері
```
Comment (nazar-pc) [inline res/translations/uk-UA/messages.ftl:124]:
```suggestion
running_farmer_farm_tooltip = Натисніть щоб відкрити в файловому менеджері
```

**Review by nazar-pc — COMMENTED** [review]
Looks good overall, but there are many places with extra ` ` before `:` (some commented), also some things were left only partially translated

File: res/translations/uk-UA/messages.ftl (hunk 6 above)
Comment (Shegyll) [reply res/translations/uk-UA/messages.ftl]: Done

**Review by nazar-pc — APPROVED** [review]
Thanks!

**Pull request merged**